build.sbt

```diff
@@ -0,0 +1,76 @@
+import sbt.Tests.{Group, SubProcess}
+import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin._
+import uk.gov.hmrc.SbtAutoBuildPlugin
+import uk.gov.hmrc.SbtArtifactory
+
+val appName: String = "tai"
+
+lazy val playSettings: Seq[Setting[_]] = Seq(routesImport ++= Seq("uk.gov.hmrc.tai.binders._", "uk.gov.hmrc.domain._"))
+
+val akkaResolver = "com.typesafe.akka"
+val akkaVersion = "2.5.23"
+val akkaHttpVersion = "10.0.15"
+lazy val dependencyOverride: Set[ModuleID] = Set(
+  akkaResolver %% "akka-stream"    % akkaVersion force (),
```

Comment:
Just an observation do we need to use force, sbt doesn't recommend it, or is this just in case!
https://www.scala-sbt.org/1.x/docs/Library-Management.html#Overriding+a+version

Reply:
It's just to force the Akka version until we upgrade to Scala 2.12, this whole override block can be removed during that process, I'll stick a note on the ticket for that.